malariagen_data/anoph/sample_metadata.py

```diff
@@ -1183,6 +1183,157 @@ def cohorts(
+        category_orders: plotly_params.category_order = None,
+        hover_name: plotly_params.hover_name = "location",
+        zoom: plotly_params.zoom = None,
+        width: plotly_params.fig_width = 800,
+        height: plotly_params.fig_height = 600,
+        show: plotly_params.show = True,
+        renderer: plotly_params.renderer = None,
+        **kwargs,
+    ) -> plotly_params.figure:
+        # Get the sample metadata.
+        df_samples = self.sample_metadata(
+            sample_sets=sample_sets,
+            sample_query=sample_query,
+            sample_query_options=sample_query_options,
+        )
+
+        # Set the location columns to use from the sample metadata.
+        location_columns = [
+            "country",
+            "admin1_iso",
+            "admin1_name",
+            "admin2_name",
+            "location",
+            "latitude",
+            "longitude",
+        ]
+
+        # Trim and dedupe the sample locations.
+        # Sort by `color` column by default, which can be overridden via category_orders.
+        df_locations = df_samples[location_columns].drop_duplicates().sort_values(color)
+
+        fig = px.scatter_mapbox(
+            df_locations,
+            lat="latitude",
+            lon="longitude",
+            mapbox_style="open-street-map",
+            zoom=zoom,
+            color=color,
+            category_orders=category_orders,
+            color_discrete_sequence=color_discrete_sequence,
+            hover_name=hover_name,
+            hover_data=location_columns,
+            width=width,
+            height=height,
+            **kwargs,
+        )
+
+        # Set the size of the markers.
+        fig.update_traces(marker=dict(size=marker_size))
+
+        if show:  # pragma: no cover
+            fig.show(renderer=renderer)
+            return None
+        else:
+            return fig
+
+    @check_types
+    @doc(
+        summary="""
+            Plot markers on a map showing sample locations
+            as a geographic scatter plot.
+        """,
+        parameters=dict(
+            kwargs="Passed through to px.scatter_mapbox().",
+        ),
+    )
+    def plot_sample_location_geo(
+        self,
+        *,
+        sample_sets: Optional[base_params.sample_sets],
+        sample_query: Optional[base_params.sample_query] = None,
+        sample_query_options: Optional[base_params.sample_query_options] = None,
+        marker_size: plotly_params.marker_size = 10,
+        color: plotly_params.color = "admin1_name",
+        color_discrete_sequence: plotly_params.color_discrete_sequence = px.colors.qualitative.Prism,
+        category_orders: plotly_params.category_order = None,
+        hover_name: plotly_params.hover_name = "location",
+        fitbounds: plotly_params.fitbounds = "locations",
+        scope: plotly_params.scope = "world",
+        width: plotly_params.fig_width = 500,
+        height: plotly_params.fig_height = 500,
```

Comment:
```suggestion
        width: plotly_params.fig_width = 800,
        height: plotly_params.fig_height = 600,
```

Suggest to use the same default width and height as the mapbox function. 500px is a bit small.